### PR TITLE
[SDK-319]: Added login page to Spring example

### PR DESCRIPTION
**Pull request by echarrod**
- Added login page to Spring Example, which required adding the application Id to the application.yml
- Added full name & isAgeVerified to example project

### CODE REVIEW

File: yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java [hunk 1]
```diff
@@ -63,21 +86,25 @@ public boolean equals(final Object o) {
...
         return accessSecurityKey != null ? accessSecurityKey.equals(that.accessSecurityKey) : that.accessSecurityKey == null;
 
     }
 
     @Override
     public int hashCode() {
+        int prime = 31;
         int result = clientSdkId != null ? clientSdkId.hashCode() : 0;
-        result = 31 * result + (accessSecurityKey != null ? accessSecurityKey.hashCode() : 0);
+        result = prime * result + (applicationId != null ? applicationId.hashCode() : 0);
```
Comment (echarrod) [inline yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java]: Couldn't see any usages of this class, and 900+ usages of the base class, so not really sure where this is used. 
I used the same format that [/blob/master/yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/Receipt.java](https://github.com/getyoti/yoti-java-sdk/blob/master/yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/Receipt.java#L160) uses

File: yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java [hunk 2]
```diff
@@ -1,30 +1,46 @@
 package com.yoti.api.examples.springboot;
 
+import com.yoti.api.client.ActivityDetails;
```
Comment (echarrod) [inline yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java]: I think IDEA auto-reordered this, is this right?

File: yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java [hunk 3]
```diff
@@ -18,6 +23,24 @@
...
+     * Gets the Yoti Application id that is provided to the client developer via Yoti dashboard.
+     *
+     * @return the Yoti Application id.
+     */
+    public String getApplicationId() {
+        return applicationId;
+    }
+
+    /**
+     * Sets the Yoti Application id that is provided to the client developer via Yoti portal.
```
Comment (qzagarese) [inline yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java]: Yoti Dashboard

File: yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java [hunk 4]
```diff
@@ -18,6 +23,24 @@
      */
     private String accessSecurityKey;
 
+    /**
+     * Gets the Yoti Application id that is provided to the client developer via Yoti dashboard.
```
Comment (qzagarese) [inline yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java]: Dashboard

File: yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java (hunk 1 above)
Comment (qzagarese) [reply yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java]: You really should let your IDE manage the generation of the hashCode method.

File: yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java [hunk 5]
```diff
@@ -63,21 +86,25 @@ public boolean equals(final Object o) {
...
+        result = prime * result + (applicationId != null ? applicationId.hashCode() : 0);
+        result = prime * result + (accessSecurityKey != null ? accessSecurityKey.hashCode() : 0);
         return result;
     }
 
     @Override
     public String toString() {
         return "YotiClientProperties{" +
-                "clientSdkId='" + clientSdkId + '\'' +
+                "applicationId='" + applicationId + '\'' +
```
Comment (qzagarese) [inline yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java]: same for toString

File: yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java (hunk 2 above)
Comment (qzagarese) [reply yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java]: not ideal, but alright.

File: yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java [hunk 6]
```diff
@@ -1,30 +1,46 @@
...
+@EnableConfigurationProperties({YotiClientProperties.class, YotiProperties.class})
 @Controller
 public class YotiLoginController {
 
     private static final Logger LOG = LoggerFactory.getLogger(YotiLoginController.class);
 
     private final YotiClient client;
 
+    @Autowired
+    private YotiClientProperties properties;
```
Comment (qzagarese) [inline yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java]: For consistency, can you inject this via constructor please?

**Pull request merged**